src/constants/common.constants.tsx

```diff
@@ -0,0 +1,335 @@
+const Paths = {
+  main: {
+    path: '/',
+    title: 'Main',
+  },
+  about: {
+    path: '/about',
+    title: 'About Us',
+  },
+  error: {
+    path: '*',
+    title: 'Error',
+  },
+  critiques: {
+    path: '/critiques',
+    title: 'Critiques',
+  },
+};
+
+const Buttons = {
+  search: 'SEARCH',
+};
+
+const Content = {
+  rating: 'Rating: ',
+  greeting: 'Enter the movie you are looking for, search by any characteristics.',
+  about:
+    "MovieSearch is a unique movie search platform. Here you'll find a movie or cartoon for every taste. We take care of adding new releases as soon as they are released. Use our search to find a movie to enjoy tonight.",
+  critiquesTitle: 'Here you can leave your review and read the reviews of other users.',
+  formTitle: 'Leave a review: ',
+  critique: {
+    nameLabel: "You'r full name: ",
+    dateLabel: 'Date viewed: ',
+    movieLabel: 'Movie title: ',
+    reviewLabel: "You'r review: ",
+    recommend: 'recommend',
+    unrecommended: "don't recommend",
+    personalDate: 'I consent to my personal data field',
+    country: 'Country of residence: ',
+    photo: 'Profile picture: ',
+    cardMovie: 'Movie: ',
+    cardRecommended: 'Recommends for viewing: ',
+    cardDate: 'Viewing date: ',
+    cardReview: 'Review: ',
+    recommendedMovie: 'yes',
+    unrecommendedMovie: 'no',
+  },
+  formSubmit: 'Send a review',
+};
+
+const InputName = {
+  name: 'name',
+  country: 'country',
+  photo: 'photo',
+  movie: 'movie',
+  date: 'date',
+  review: 'review',
+  personalDate: 'personal',
+};
+
+const ImageFormats = '.jpg,.jpeg,.png';
+
+const Pages = {
+  main: 'Main',
+  about: 'About Us',
+  error: '404 Page not found',
+  critiques: 'Critiques',
+};
+
+const LocalStorageKeys = {
+  search: 'search',
+};
+
+const Countries = [
+  'Afghanistan',
```

Comment:
maybe better to use some npm package for countries list?

Reply:
accidentally run into a not very smart inspector who will decide that I have connected something forbidden. Must use next time)